src/app/tests/crud/test_crud_generation_template.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.models.generation_template import GenerationTemplate
+from app.crud.crud_generation_template import generation_template
+from app.schemas.generation_template import GenerationTemplateUpdate
+from app.tests.utils.models import create_generation_template
+
+pytestmark = pytest.mark.asyncio
+
+
+async def test_crud_generation_template_create(database: AsyncSession):
+    generation_template_obj = await create_generation_template(db=database)
+    assert generation_template_obj is not None
+    assert isinstance(generation_template_obj, GenerationTemplate)
```

Comment:
Please also check the that the object has arrived in the database. (And not just that I return the request body and still get a pass :wink:)

Reply:
Addressed